### PR TITLE
A number of misc version bumps (WIP!)

### PR DESCRIPTION
**Pull request by victoredwardocallaghan**
Signed-off-by: Edward O'Callaghan eocallaghan@alterapraxis.com

### CODE REVIEW

**Comment by lethalman** [comment]
I don't use irssi, but jumping from a 2013 tag to an untagged revision is not good practice. What about using a release instead, and use fetchFromGithub: https://github.com/irssi/irssi/releases.

**Comment by pSub** [comment]
I don't see the commit for `irssi-otr` anymore. However the commit for `bullet` does not change the sha256 hash.

File: pkgs/development/interpreters/php/5.4.nix [hunk 1]
```diff
@@ -9,7 +9,7 @@ in
 
 composableDerivation.composableDerivation {} ( fixed : let inherit (fixed.fixed) version; in {
 
-  version = "5.4.35";
+  version = "5.6.5";
```
Comment (lethalman) [inline pkgs/development/interpreters/php/5.4.nix]: The file is called 5.4. And you should not bump php like that in the same derivation, because it's not supposed to be backward compatible.

Comment (victoredwardocallaghan) [inline pkgs/development/interpreters/php/5.4.nix]: yep good catch, thx.

**Comment by pSub** [comment]
As a side note: I find it hard to follow, if a pull requests contains so many different things and changes frequently. I would prefer separate PRs for the updates.

**Comment by victoredwardocallaghan** [comment]
Yes, sorry I am abusing the build host at the moment until I have something more suitable setup locally at the end of the week.

**Comment by lethalman** [comment]
Closing this, please open a PR for each of the update. Thanks.
